deploy/ocp/manifests/0.7.2/0000_30_00-namespace.yaml

```diff
@@ -6,3 +6,5 @@ metadata:
   name: openshift-operator-lifecycle-manager
   labels:
     openshift.io/run-level: "1"
+  annotations:
+    openshift.io/node-selector: ""
```

Comment:
@ecordell do we need to update every namespace.yaml for all releases?

Reply:
We should just need latest